hail/python/hailtop/batch/__init__.py

```diff
@@ -4,14 +4,18 @@
 from .batch_pool_executor import BatchPoolExecutor
 from .backend import LocalBackend, ServiceBackend, Backend
 from .utils import BatchException
+from .functions import _combine, concatenate, plink_merge
 
 __all__ = ['Batch',
            'LocalBackend',
            'ServiceBackend',
            'Backend',
            'BatchException',
            'BatchPoolExecutor',
-           'genetics'
+           '_combine',
```

Comment:
I'm inclined to not expose this at the top-level if its undocumented. Users can always access it as `hailtop.batch.functions._combine`.